### PR TITLE
Promote Qdrant extension to native

### PR DESCRIPTION
**Pull request by aldettinger**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: extensions/qdrant/runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+		<camel.quarkus.jvmSince>3.9.0</camel.quarkus.jvmSince>
+		<camel.quarkus.nativeSince>3.9.0</camel.quarkus.nativeSince>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.camel.quarkus</groupId>
+			<artifactId>camel-quarkus-core</artifactId>
+		</dependency>
+		<dependency>
```
Comment (aldettinger) [inline extensions/qdrant/runtime/pom.xml:45]: @jamesnetherton @ppalaga  A dependency on camel-quarkus-grpc is needed to have the native tests run locally.
Would you have any idea to integrate this cleanly in the source tree ? We need  kind of support project ?

Comment (jamesnetherton) [reply extensions/qdrant/runtime/pom.xml:45]: Adding `quarkus-grpc-common` should be enough, no?

Comment (aldettinger) [reply extensions/qdrant/runtime/pom.xml:55]: Indeed, it might be sufficient. There is an issue but it's not related to qdrant :+1:

**Comment by aldettinger** [comment]
Closing in favor of #5948

**Pull request closed without merging**